supabase/migrations/20250312121737_data-planes-events-column-constraints.sql

```diff
@@ -0,0 +1,19 @@
+begin;
+
+-- Populate these new columns. Note that the legacy data plane needs some
+-- special handling because it doesn't follow the normal naming convention.
+-- The substr call is to strip off the 'ops/dp/' prefix.
+update public.data_planes
+set
+    ops_l1_events_name = case when data_plane_name = 'ops/dp/public/gcp-us-central1-c1'
+        then 'ops/rollups/L1/public/gcp-us-central1-c1/events'
```

Comment:
why does cronut need a special case? you say it doesn't follow the naming convention, but that doesn't seem to matter here?

Reply:
Yeah, I think I'd initially thought to stick with the prior existing naming convention for this data plane (which would have resulted in `ops.us-central1.v1/events-L1`), but then changed my mind. This portion of the migration has already been run, and has already established the cronut events collection name. It's copied into this migration so that it can update the couple of data planes that were created since running the initial migration.